### PR TITLE
Make measurement system units changeable.

### PR DESCRIPTION
**Pull request by LinusThorsell**
This PR contains the following additions to the Advanced User Settings menu:
bild

This allows the user to switch between different measurement systems/units for GPS speed and GPS/Baro Altitude in the Legend Screen.

Default:
bild

Imperical Example:
bild

### CODE REVIEW

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:870]: Missing semicolon

Comment (LinusThorsell) [reply js/flightlog_fields_presenter.js]: Resolved.

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Pull request merged**
